### PR TITLE
[feature] restructure-clean-up-project

### PR DESCRIPTION
**Pull request by gardlt**
* updating file-structure
* updating logs

**What is the purpose of this pull request?**:

**What issue does this pull request address?**: Fixes #

**Notes for reviewers to consider**:

**Specific reviewers for pull request**:

### CODE REVIEW

File: armada/handlers/armada.py [hunk 1]
```diff
@@ -37,15 +58,24 @@ def find_release_chart(self, known_releases, name):
             if chart_name == name:
                 return chart, values
 
+    def pre_flight_checks(self):
+        for ch in self.config['armada']['charts']:
+            location = ch.get('chart').get('source').get('location')
+            ct_type = ch.get('chart').get('source').get('type')
+
+            if ct_type == 'git' and not git.check_available_repo(location):
+                raise ValueError(str("Invalide Url Path: " + location))
```
Comment (mattmceuen) [inline armada/handlers/armada.py]: The spelling of this error message is invalide

File: docs/source/development/getting-started.rst [hunk 2]
```diff
@@ -5,11 +5,22 @@ Development
...
+
+1. Fork the [repo](http://github.com/att-comdev/armada)
+2. After making changes push to repo and build from forked repo
 
 .. code-block:: bash
 
-   docker run quay.io/attcomdev/armada:latest
+    export repo="https://github.com/<forked-repo>/armada.git"
+    export branch="<branch>"
+    docker build . -p 8000:8000 -v ${HOME}/.kube/config:/root/.kube/config -t quay.io/attcomdev/armada:latest --build-arg REPO=$repo --build-arg VERSION=$branch
```
Comment (drewwalters96) [inline docs/source/development/getting-started.rst]: This should be `docker run`

Comment (alanmeadows) [inline docs/source/development/getting-started.rst]: For some reason I wasn't alerted to this pull request.  If you want to update this documentation ask Alex, I will merge this.

Comment (drewwalters96) [inline docs/source/development/getting-started.rst]: @alanmeadows After a discussion with Alex this morning, please merge. I will update the documentation afterwords.

**Pull request merged**